pkg/symbolizer/symbolizer.go

```diff
@@ -33,6 +34,11 @@ import (
 
 type Symbolizer struct {
 	logger log.Logger
+	// attempts counts the total number of symbolization attempts.
+	attempts prometheus.Counter
+	// errors counts the total number of symbolization errors, partitioned by an error reason
+	// such as failure to fetch unsymbolized locations.
+	errors *prometheus.CounterVec
```

Comment:
I used two counters, although one can do the job. Please let me know which option you prefer.

Reply:
This looks perfect :+1: